docs/kb/error-codes-messages.md

```diff
@@ -1627,6 +1675,150 @@ requests.
 
 **Solution**: Reset your password at https://pi.pardot.com/user/passwordReset
 
+## [](#error-code-132)Error Code: 132
+
+```
+<rsp stat="fail" version="1.0">
+    <err code="132">Prospect is archived</err>
+</rsp>
+```
+
+**Problem**: The prospect cannot be read because it is in the recycle bin.
+
+**Solution**: Undelete the prospect via the UI or choose another prospect.
+
+## [](#error-code-133)Error Code: 133
+
+```
+<rsp stat="fail" version="1.0">
+    <err code="133">Up to 2 sort by options are allowed.</err>
+</rsp>
+```
+
+**Problem**: Too many columns were passed into the sort parameter.
+
+**Solution**: Limit sort columns to 2 or fewer.
```

Comment:
Does this construction make sense in context? Should it be something like:
 - Use 2 or fewer sort by options
- Use 2 or fewer sort by columns 
? There seems to be bit of a disconnect between the error message and the solution text. Consider the terms the reader will understand here.